Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM alpine:latest
+
+# Install dependencies
+#RUN apk add --no-cache --update postfix ca-certificates socat acme.sh bash && \
+RUN apk add --no-cache --update postfix ca-certificates
+
+# Expose smtp port
+EXPOSE 25
+
+#postfix config
+RUN postconf -e mynetworks=0.0.0.0/0
+RUN postconf -e "maillog_file=/dev/stdout"
+
+#add script execution
+#https://contrid.net/server/mail-servers/postfix-catch-all-pipe-to-script
+RUN echo "@student.dhbw-mannheim.de nobody@student.dhbw-mannheim.de" > /etc/postfix/virtual_aliases
+RUN echo "student.dhbw-mannheim.de  zimbrawebtransport:" > /etc/postfix/transport
```

Comment:
If this is changed to
```
RUN echo "" > /etc/postfix/virtual_aliases
RUN echo "* zimbrawebtransport:" > /etc/postfix/transport
```
all mails will be forwared to our transport agent instead of just mails to @student.dhbw-mannheim.de

Reply:
Oh right, never tried that...

Reply:
I didnt notice that, as postfix reverts to sending directly and `@student.dhbw-mannheim.de` has no DMARC or DKIM.